docs/sources/setup-grafana/set-up-https.md

```diff
@@ -0,0 +1,277 @@
+
+To generate certificates using `certbot`, complete the following steps: 
+
+1. Ensure that port `80` traffic is permitted by applicable firewall rules.
+
+1. Run the following command to generate certificates:
+
+   ```bash
+   $ sudo certbot certonly --standalone
+   
+   Saving debug log to /var/log/letsencrypt/letsencrypt.log
+   Enter email address (used for urgent renewal and security notices)
+   (Enter 'c' to cancel): me@mysite.com
+
+   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   Please read the Terms of Service at
+   https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf. You must
+   agree in order to register with the ACME server. Do you agree?
+   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   (Y)es/(N)o: y
+   
+   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   Would you be willing, once your first certificate is successfully issued, to
+   share your email address with the Electronic Frontier Foundation, a founding
+   partner of the Let’s Encrypt project and the non-profit organization that
+   develops Certbot? We’d like to send you email about our work encrypting the web,
+   EFF news, campaigns, and ways to support digital freedom.
+   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   (Y)es/(N)o: n
+   Account registered.
+   Please enter the domain name(s) you would like on your certificate (comma and/or
+   space separated) (Enter 'c' to cancel): subdomain.mysite.com
+   Requesting a certificate for subdomain.mysite.com
+   
+   Successfully received certificate.
+   Certificate is saved at: /etc/letsencrypt/live/subdomain.mysite.com/fullchain.pem
+   Key is saved at:         /etc/letsencrypt/live/subdomain.mysite.com/privkey.pem
+   This certificate expires on 2023-06-20.
+   These files will be updated when the certificate renews.
+   Certbot has set up a scheduled task to automatically renew this certificate in the background.
+   
+   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   If you like Certbot, please consider supporting our work by:
+   * Donating to ISRG / Let’s Encrypt:   https://letsencrypt.org/donate
+   * Donating to EFF:                    https://eff.org/donate-le
+   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   ```
+
+#### Set up symlinks to Grafana
+
+Symlinks are like pointers on your filesystem that allow us to refer to the LetsEncrypt files, from within the `/etc/grafana` directory.  The reason we use symlinks to the "live" files instead of copying the files is to permit you to later refresh or re-request updated certificates from LetsEncrypt using `certbot`, without needing to change any Grafana configuration.  
```

Comment:
```suggestion
Symlinks are like pointers on your filesystem that allow us to refer to the LetsEncrypt files, from within the `/etc/grafana` directory.  The reason we use symlinks to the "live" files instead of copying the files is to permit you to later refresh or re-request updated certificates from LetsEncrypt using `certbot`, without needing to change any Grafana configuration.  

Symbolic links, also known as symlinks, enable you to create pointers to existing LetsEncrypt files in the `/etc/grafana` directory. By using symlinks rather than copying files, you can use `certbot` to refresh or request updated certificates from LetsEncrypt without the need to reconfigure the Grafana settings.
```